scripts/step.sh

```diff
@@ -44,16 +44,16 @@ test_data_dir=${work_dir}/tests/$testname
 
 ## Step8: run main file to generate ground result
 # positive case
-python ${script_dir}/main_immuno.py --mutation_mode ref --output_dir ${test_data_dir} --ann_path ${test_data_dir}/data/test1pos.gtf --splice_path ${test_data_dir}/data/posgraph/spladder/genes_graph_conf3.merge_graphs.pickle --ref_path ${test_data_dir}/data/test1pos.fa --count_path ${test_data_dir}/data/posgraph/spladder/genes_graph_conf3.merge_graphs.count.hdf5 --samples test1pos --debug --vcf_path ${test_data_dir}/data/test1pos.vcf --maf_path ${test_data_dir}/data/test1pos.maf
```

Comment:
I assume the flag is no longer used because it can be inferred from whether `--vcf_path` and `--maf_path` flags are provided. This shifts the responsibility to the user to check their inputs, but I assume this is fine.